java/java.api.common/src/org/netbeans/modules/java/api/common/project/ui/MultiModuleNodeFactory.java

```diff
@@ -557,12 +561,40 @@ public String getHtmlDisplayName() {
                         null,
                         SystemAction.get(FileSystemAction.class ),
                         null,
-                        SystemAction.get(ToolsAction.class )
+                        SystemAction.get(ToolsAction.class ),
+                        //Added new action to delete module
```

Comment:
From the name of the action the intention is clear, new is relative (in a few years it is not new anymore) -> please remove the comment.